src/crimson/os/seastore/extent_placement_manager.h

```diff
@@ -397,8 +358,6 @@ class ExtentPlacementManager {
     return devices[std::rand() % devices.size()];
   }
 
-  Cache& cache;
```

Comment:
I think we'll have to deal with splitting/merging extents when doing delayed extent allocation, which means we'll have to access cache from EPM. Moving EPM under cache may lead to mutual references between EPM and Cache in the future. Is this acceptable?

Reply:
my colleague is currently working on splitting/merging extents: https://github.com/zhscn/ceph/commits/wip-split-extent

Reply:
see https://github.com/ceph/ceph/pull/44809#discussion_r800662230